lib/dialects/postgres/query-generator.js

```diff
@@ -417,8 +417,13 @@ var QueryGenerator = {
 
   addLimitAndOffset: function(options) {
     var fragment = '';
-    if (options.limit) fragment += ' LIMIT ' + this.escape(options.limit);
-    if (options.offset) fragment += ' OFFSET ' + this.escape(options.offset);
+
+    if (typeof options.limit !== "undefined" && options.limit !== null) {
+      fragment += ' LIMIT ' + this.escape(options.limit);
+    }
+    if ( typeof options.limit !== "undefined" && options.limit !== null) {
```

Comment:
Actually not since default offset is 0, but I did it to keep a kind of consistency.